content/administration/install/deploy.rst

```diff
@@ -388,20 +388,109 @@ The solutions to support livechat/motifications in a WSGI application are:
   with ``/longpolling/`` to
   :option:`the longpolling port <odoo-bin --longpolling-port>`.
 
+.. _streaming:
+
+Serving Static Files and Attachments
+====================================
+
+For development convenience, Odoo directly serves all static files and attachments in its modules.
+This may not be ideal when it comes to performances, and static files should generally be served by
+a static HTTP server.
+
 Serving Static Files
-====================
+--------------------
+
+Odoo static files live in each module's ``static/`` folder, so static files can be served by
```

Comment:
- "live"?. Maybe better with "are located"
- consider using the ``file`` directive for filepaths (see https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#spice-up-your-writing-with-specialized-directives and apply the same to the rest of your diff)